crates/proto/src/rr/domain/label.rs

```diff
@@ -409,6 +480,12 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_from_ascii_adversial_utf8() {
+        let expect_err = Label::from_ascii("🦀");
+        assert!(expect_err.is_err());
+    }
+
```

Comment:
Small regression test, did a quick-check on `from_ascii` call look like it was not done before maybe i am wrong.